### ISSUE TITLE
[Accessibility] Talkback doesn't announce anything about Slider when focused

### ISSUE THREAD

**Issue by WillowWisp** [flutter/flutter#94326]
Similar issue: https://github.com/flutter/flutter/issues/79528
I saw that there was a merged PR for this issue https://github.com/flutter/engine/pull/27992. Although, the bug somehow is still there even on **beta channel** (please check out my flutter doctor below). 

## Steps to Reproduce

1. clone the slider interactive app from the link: https://api.flutter.dev/flutter/material/Slider-class.html
2. run the app and enable talkback
3. focus on the slider (by either swiping left/right or touching the slider)

**Expected results:**
Talkback should announce the Slider on a11y focused.

**Actual results:**
Talkback doesn't announce anything unless we increase/decrease the Slider.

<details>
  <summary>flutter doctor -v</summary>

```
[✓] Flutter (Channel beta, 2.8.0-3.2.pre, on macOS 11.6 20G165 darwin-x64, locale en)
    • Flutter version 2.8.0-3.2.pre at /Users/nguyenthang/Desktop/ThangDev/FlutterSDK/flutter_macos_2.8.0-3.2.pre-beta
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 2901cd7208 (6 days ago), 2021-11-18 11:15:09 -0800
    • Engine revision bcc2b7f12c
    • Dart version 2.15.0 (build 2.15.0-268.18.beta)

[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at /Users/nguyenthang/Library/Android/sdk
    • Platform android-31, build-tools 31.0.0
    • ANDROID_HOME = /Users/nguyenthang/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7281165)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 13.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • CocoaPods version 1.11.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2020.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7281165)

[✓] VS Code (version 1.62.3)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.28.0

[✓] Connected device (2 available)
    • CPH2219 (mobile) • a2e11d62 • android-arm64  • Android 11 (API 30)
    • Chrome (web)     • chrome   • web-javascript • Google Chrome 96.0.4664.55

• No issues found!
```

</details>

Thanks, Flutter team!

**Comment by mamuseferha** [comment]
Hi @WillowWisp, can you please test this on the latest master channel to see if it is reproducible. 

This issue seems to only occur on specific devices. I tested on a Samsung S10+ and it is not reproducible.

<details>
<summary>flutter doctor -v</summary>

```bash
[✓] Flutter (Channel stable, 2.5.3, on macOS 11.6.1 20G224 darwin-x64, locale en-US)
    • Flutter version 2.5.3 at /Users/flo/development/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 18116933e7 (6 weeks ago), 2021-10-15 10:46:35 -0700
    • Engine revision d3ea636dc5
    • Dart version 2.14.4

[!] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at /Users/flo/Library/Android/sdk
    • Platform android-31, build-tools 31.0.0
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7281165)
    ! Some Android licenses not accepted.  To resolve this, run: flutter doctor --android-licenses

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 13.1, Build version 13A1030d
    • CocoaPods version 1.11.0

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2020.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7281165)

[✓] VS Code (version 1.62.3)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.28.0

[✓] Connected device (4 available)
    • SM G975U (mobile)            • R58M95MA95E   • android-arm64  • Android 11 (API 30)
    • sdk gphone64 x86 64 (mobile) • emulator-5554 • android-x64    • Android 12 (API 31) (emulator)
    • macOS (desktop)              • macos         • darwin-x64     • macOS 11.6.1 20G224 darwin-x64
    • Chrome (web)                 • chrome        • web-javascript • Google Chrome 93.0.4577.63

! Doctor found issues in 1 category.
```
```bash
[✓] Flutter (Channel master, 2.6.0-12.0.pre.847, on macOS 11.6.1 20G224 darwin-x64, locale en-US)
    • Flutter version 2.6.0-12.0.pre.847 at /Users/flo/fvm/versions/master
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 8b1cf766e7 (18 hours ago), 2021-11-28 10:47:02 -0500
    • Engine revision 4536092a7f
    • Dart version 2.16.0 (build 2.16.0-55.0.dev)
    • DevTools version 2.8.0

[!] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at /Users/flo/Library/Android/sdk
    • Platform android-31, build-tools 31.0.0
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7281165)
    ! Some Android licenses not accepted.  To resolve this, run: flutter doctor --android-licenses

[✓] Xcode - develop for iOS and macOS (Xcode 13.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • CocoaPods version 1.11.0

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2020.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7281165)

[✓] VS Code (version 1.62.3)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.28.0

[✓] Connected device (4 available)
    • SM G975U (mobile)            • R58M95MA95E   • android-arm64  • Android 11 (API 30)
    • sdk gphone64 x86 64 (mobile) • emulator-5554 • android-x64    • Android 12 (API 31) (emulator)
    • macOS (desktop)              • macos         • darwin-x64     • macOS 11.6.1 20G224 darwin-x64
    • Chrome (web)                 • chrome        • web-javascript • Google Chrome 93.0.4577.63

! Doctor found issues in 1 category.
```

</details>

Thank you

**Comment by WillowWisp** [comment]
Hi @mamuseferha, I can confirm that this has been fixed in flutter 2.8.0, so I guess we can close this now.
Thanks for your help!
